students/bishal_gupta/lesson03/mailroom.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+"""
+Created on Sat Nov  2 14:12:38 2019
+
+@author: Bishal.Gupta
+"""
+
+import sys  # imports go at the top of the file
+
+donor_db = [("William Gates, III", [653772.32, 12.17]),
```

Comment:
Main problem with your data structure is how to update the list of donations. You COULD have a list of lists, where the first element of the list if ALWAYS the name...